firecloud/fiss.py

```diff
@@ -859,23 +859,42 @@ def attr_get(args):
     a special __header__ entry is optionally added to the result. '''
 
     if args.entity_type and args.entity:
-        r = fapi.get_entity(args.project, args.workspace, args.entity_type, args.entity)
+        if args.entity_type == "ref":       # return referenceData (b37 or hg38) attributes
+            r = fapi.get_workspace(args.project, args.workspace, fields="workspace.attributes")
+            fapi._check_response_code(r, 200)
+            all_attrs = r.json()['workspace']['attributes']
+            # check for referenceData (b37 or hg38) in workspace
+            attrs = {k:all_attrs[k] for k in all_attrs if re.search(f'referenceData_', k)}
+            if not attrs:
+                print(f"There are no reference data available in workspace. Load a reference and try again: hg38 or b37")
+            else:
+                attrs = {k:all_attrs[k] for k in all_attrs if re.search(f'referenceData_{args.entity}', k)}
+                # if the chosen referenceData is not in workspace
+                if not attrs:
+                    print(f"The given reference, {args.entity}, is not in workspace. Try another option: hg38 or b37")
```

Comment:
Same as above (please no fstrings), also perhaps add a search for all unique references based on prefixes and return those here. This should be followed by a `return 1` so FISS knows it's an error.